src-ts/lib/work-provider/work-functions/work-factory/work.factory.ts

```diff
@@ -124,15 +126,17 @@ function buildFormDataDesign(formData: any): any {
         },
         inspiration: {
             title: 'Inspiration',
-            value: formData.inspiration?.map((item: any) => `${item.website?.value} ${item.feedback?.value}`),
+            value: formData.inspiration?.map((item: any) => `${item.website?.value} ${item.feedback?.value}`)
```

Comment:
let's add a line break before the `?.map` to make it a little more clear

Reply:
Today I learned you can line break on `?.map`!